src/signalrclient/microsoft-signalr-config.in.cmake

```diff
@@ -0,0 +1,11 @@
+include(CMakeFindDependencyMacro)
```

Comment:
What does this file do?

Reply:
It lets us add dependencies, otherwise when you resolve microsoft-signalr, you don't bring in the other dll(s) I believe.